packages/ia-user-list/README.md

```diff
@@ -0,0 +1,66 @@
+![Build Status](https://github.com/internetarchive/iaux-typescript-wc-template/actions/workflows/ci.yml/badge.svg) [![codecov](https://codecov.io/gh/internetarchive/iaux-typescript-wc-template/branch/main/graph/badge.svg?token=ZOYRJ2BV9W)](https://codecov.io/gh/internetarchive/iaux-typescript-wc-template)
```

Comment:
these don't work here. I don't want anything that doesn't work on merge.. please remove these badges & other references to config that do not work in mono repo.